src/server.cpp

```diff
@@ -1101,15 +1102,17 @@ void Server::Receive(float timeout)
 		pkt.clear();
 		peer_id = 0;
 		try {
+			// Round up since the target step length is the minimum step length,
+			// we only have millisecond precision and we don't want to busy-wait by calling
+			// ReceiveTimeoutMs(.., 0) repeatedly.
+			// Also be sure we don't wait if the remaining time is zero/negative.
```

Comment:
I don't think this last line makes sense if the reader hasn't seen the std::max alternative.
(`remaining_time_us()` is always non-negative and receive with timeout 0 is fine.)
```suggestion
```